pkg/kubelet/cm/topologymanager/policy.go

```diff
@@ -83,13 +89,15 @@ func filterProvidersHints(providersHints []map[string][]TopologyHint) [][]Topolo
 			if len(hints[resource]) == 0 {
 				klog.Infof("[topologymanager] Hint Provider has no possible NUMA affinities for resource '%s'", resource)
 				allProviderHints = append(allProviderHints, []TopologyHint{{nil, false}})
+				lackingResources = append(lackingResources, resource)
+				err = fmt.Errorf("not enough '%s' to allocate a pod", strings.Join(lackingResources, ", "))
```

Comment:
I'm not sure in general about this specific log, but this can be done outside the loop, right before we return

Reply:
Thanks for diving into the PR! The idea behind it was to create slice of all missing resources and return it entirely with ``TopologyAffinityError``, which could be truly helpful for the ones not familiar with ``Scope`` and the ``TopologyManager`` itself.